man/last.Rd

```diff
@@ -3,36 +3,47 @@
 \alias{last}
 \title{ First/last item of an object }
 \description{
-Returns the first/last item of a vector or list, or the first/last row of a data.frame
-or data.table. The main difference to head/tail is that the default for \code{n} is 1
-rather than 6.
+Returns the first/last item of a vector, list, \code{data.frame} or \code{data.table}. The main difference
+to head/tail is that the default for \code{n} is 1 rather than 6.
```

Comment:
"and support for and support for skipping missing entries"

Reply:
done